examples/layers/preprocessing/bounding_box/random_translation_demo.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+random_flip_demo.py shows how to use the RandomFlip preprocessing layer for
+object detection.
+"""
+import demo_utils
+import tensorflow as tf
+
+from keras_cv.layers import preprocessing
+
+
+def main():
```

Comment:
Could you please include a screenshot of the output of this demo in the PR description

Reply:
Added the output from the demo file. It's not the best bounding box plot but should give an idea.

Reply:
Updated to a better screenshot.